### ISSUE TITLE
WSG Roof Mounting

### ISSUE THREAD

**Issue by jakeoff** [Atlantiss/NetherwingBugtracker#4651]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: You can mount/remain mounted on both the Alliance/Horde roof in WSG.

**Current behaviour**: You can mount/remain mounted on both the Alliance/Horde roof in WSG.

**Expected behaviour**: Roofs should be an off-limit area for mounting. You should be kicked off your mount as you step on to the roof, and you should not be able to mount while already there.

**Server Revision**:

**Comment by Lockanloads** [comment]
Yes, this is not so unfair because players on both  sides can mount to the roof and then access the flag room with mount on.
 Its mostly unfair to rogues because they had the strategic sprint advantage that is then consequently slightly reduced.

**BUT** its **OP** for Shamans and  especially Druids because they can carry the flag without unforming and gaining 40% movespeed (55% for a feral with 4 set bonus), about 3-4 seconds lesser than a regular 7-8 seconds run to the outside at the most dangerous point for the player.

This basically means that as players will use this, droods/sham teams will statistically win more.

PS : When a player with mount in EFC goes to the tunnel, the mount is canceled. Wrong outside/inside calculation at first time.

**Comment by brutalaty** [comment]
![rooftop](https://github.com/Atlantiss/NetherwingBugtracker/assets/6793397/abbd5ca3-4832-45a0-9548-41728c91908c)

Can confirm, still a bug.
